dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -61,16 +61,15 @@
    * Returns list of coordinates of Maven artifacts if specified; otherwise an empty list. Example
    * element: {@code com.google.cloud:google-cloud-bigtable:0.66.0-alpha}
    */
-  abstract ImmutableList<String> getMavenCoordinates();
+  abstract ImmutableList<String> getMavenCoordinatesList();
```

Comment:
Can we better distinguish this from the BOM option? That's Maven coordinates too, just for a BOM artifact instead of a list of JAR artifacts.

The three options for specifying a class path are

1. one BOM artifact
2. a list of non-BOM JAR artifacts
3. a list of paths to jar files

What about

1. `--bom`, `getBomArtifact()`
2. `--jar_artifacts`, `getJarArtifacts()`
3. `--jar_paths`, `getJarPaths()`

If we want to be clever (which we may not), we could combine (2) and (3) into `--jars` and figure out at runtime which are paths and which are Maven coordinates.

```
static_linkage_checker --bom=some.group:some-bom:1.0
static_linkage_checker --jars=some.group:some-artifact:1.0,/path/to/a.jar,/path/to/another.jar,other.group:other-artifact:2.1
```

Reply:
I'd like to keep these three separated as designed in [Static Linkage Checker: Input](https://github.com/GoogleCloudPlatform/cloud-opensource-java/tree/master/dependencies#input).

I updated the properties as "bom", "artifacts", "jarFiles" by removing "coordinates" and "list". Do they look better?